app/controllers/jobseekers/base_controller.rb

```diff
@@ -9,6 +9,8 @@ def storable_location?
   end
 
   def store_jobseeker_location!
+    return if request.referrer && URI(request.referrer).path == "/jobseekers/sign-in"
```

Comment:
I think it's more often spelt `referer`, due to a historic misspelling when the protocol was agreed...

Bear in mind not all browsers will set the referer header, so this wont' work for everyone.